tests/platform_tests/api/conftest.py

```diff
@@ -54,8 +68,21 @@ def start_platform_api_service(duthosts, enum_rand_one_per_hwsku_hostname, local
         duthost.command('docker exec -i pmon supervisorctl reread')
         duthost.command('docker exec -i pmon supervisorctl update')
 
-        res = localhost.wait_for(host=dut_ip, port=SERVER_PORT, state='started', delay=1, timeout=10)
-        assert res['failed'] is False
+        res = localhost.wait_for(host=dut_ip, port=SERVER_PORT, state='started', delay=1, timeout=5)
+        if res['failed'] is False:
+            return False
+
+    return True
```

Comment:
@vdahiya12 we are not using the return value, so we don't need to return anything

Reply:
wait_until seems to expect return True if function correctly returns 

https://github.com/sonic-net/sonic-mgmt/blob/a1ba0400d4b6c6b148b3ac34d1eda3378fe78e43/tests/common/utilities.py#L115C3-L116C63